### PR TITLE
46 replace type function and add presssequentially support

### LINKED ISSUE #46: Replace Type function and add pressSequentially support
Since 1.38 the following methods were deprecated: 
[page.type()](https://playwright.dev/docs/api/class-page#page-type), 
[frame.type()](https://playwright.dev/docs/api/class-frame#frame-type), 
[locator.type()](https://playwright.dev/docs/api/class-locator#locator-type) and 
[elementHandle.type()](https://playwright.dev/docs/api/class-elementhandle#element-handle-type). 
Please use: 
[locator.fill()](https://playwright.dev/docs/api/class-locator#locator-fill) instead which is much faster. Use [locator.pressSequentially()](https://playwright.dev/docs/api/class-locator#locator-press-sequentially) only if there is a special keyboard handling on the page, and you need to press keys one-by-one.

### PR DESCRIPTION
**Pull request by csczygiol**
Fix #46

### CODE REVIEW

File: docs/screenplay_elements/Web/Ability/browse_the_web.md [hunk 1]
```diff
@@ -111,12 +111,14 @@ public async hover(selector: Selector, options?: SelectorOptions & { modifiers?:
 #### press
 
 ```typescript
-public async press(input: string): Promise<void>;
+public async type(selector: Selector, input: string, options?: SelectorOptions): Promise<void>;
```
Comment (munichbughunter) [inline docs/screenplay_elements/Web/Ability/browse_the_web.md:114]: Seems to be wrong. Why is it type?

File: docs/screenplay_elements/Web/Ability/browse_the_web.md [hunk 2]
```diff
@@ -111,12 +111,14 @@ public async hover(selector: Selector, options?: SelectorOptions & { modifiers?:
...
 ```
 
 - **Description:** Presses the specified key(s) on the keyboard.
 - **Parameters:**
   - `input` - The key(s). Multiple keys can be pressed by concatenating with "+".
+  - `selector` - The selector of the source element.
```
Comment (munichbughunter) [inline docs/screenplay_elements/Web/Ability/browse_the_web.md:120]: Sort it right. 
- `selector`- ....
- `input`- ....
- `options`- ...

**Review by munichbughunter — CHANGES_REQUESTED** [review]
I miss a test for press with mode 'key' and I miss a test for press with mode 'sequentially'.
Please add them.

**Comment by csczygiol** [comment]
> I miss a test for press with mode 'key' and I miss a test for press with mode 'sequentially'.

> Tests got added

File: docs/screenplay_elements/Web/Ability/browse_the_web.md (hunk 2 above)
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Ability/browse_the_web.md:119]: Done

**Review by munichbughunter — APPROVED** [review]
From my side it looks good.

**Pull request merged**